src/values.ts

```diff
@@ -0,0 +1,50 @@
+import { OptionsValues } from "./types";
+
+export interface DateInputValue {
```

Comment:
why not moving to `./types` file ?

Reply:
It's easier to export all these out of a single file via `index`, but we probably don't want to export all `types` out of index. So I would have to move them and then re-export them from `index`

Also generally types are being used for more generic multi purpose types, I don't mind doing that but if someone adds additional types they will have to remember to rexport them.